### PR TITLE
http transport support to replace amq

### PR DESCRIPTION
**Pull request by aneeshkp**
add service to expose http transport endpoints
This addes http transport protocol for ptp events
Signed-off-by: Aneesh Puttur <aneeshputtur@gmail.com>

### CODE REVIEW

File: bindata/linuxptp/ptp-daemon.yaml [hunk 1]
```diff
@@ -173,6 +173,28 @@ spec:
...
+    app: linuxptp-daemon
+  name:  ptp-event-publisher-service
+  namespace: openshift-ptp
+spec:
+  selector:
+    app: linuxptp-daemon
+  clusterIP: None
+  ports:
+    - name: publisher-port
+      port: 9086
```
Comment (josephdrichard) [inline bindata/linuxptp/ptp-daemon.yaml:194]: Why port 9086?

**Review by josephdrichard — COMMENTED** [review]
Is there a change to the bundle required?

File: bindata/linuxptp/ptp-daemon.yaml (hunk 1 above)
Comment (aneeshkp) [reply bindata/linuxptp/ptp-daemon.yaml]: changed it to 9043, its the port that is running default publisher service (HTP  event Publisher service)
I just picked one that would conflict with the one most used

**Comment by aneeshkp** [comment]
> Is there a change to the bundle required?

Not for this change I think . The code change is in side car and I am adding only new service to ptp operator which is deployed via  dindata/ folder. (Old method )  and not part of CSV

**Pull request merged**